script/c100217005.lua

```diff
@@ -0,0 +1,89 @@
+--EM五虹の魔術師
+--Performapal Five-Arc Magician
+--Scripted by Eerie Code
+function c100217005.initial_effect(c)
+	aux.EnablePendulumAttribute(c)
+	--scale
+	local e1=Effect.CreateEffect(c)
+	e1:SetCategory(CATEGORY_SPECIAL_SUMMON)
+	e1:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_TRIGGER_O)
+	e1:SetCode(EVENT_SSET)
+	e1:SetRange(LOCATION_GRAVE)
+	e1:SetCondition(c100217005.pencon)
+	e1:SetTarget(c100217005.pentg)
+	e1:SetOperation(c100217005.penop)
+	c:RegisterEffect(e1)
+	--splimit
+	local e2=Effect.CreateEffect(c)
+	e2:SetType(EFFECT_TYPE_FIELD)
+	e2:SetRange(LOCATION_PZONE)
+	e2:SetCode(EFFECT_CANNOT_SPECIAL_SUMMON)
+	e2:SetProperty(EFFECT_FLAG_PLAYER_TARGET+EFFECT_FLAG_CANNOT_DISABLE+EFFECT_FLAG_CANNOT_NEGATE)
+	e2:SetTargetRange(1,0)
+	e2:SetTarget(c100217005.splimit)
+	c:RegisterEffect(e2)
+	--prevent attack
+	local e3=Effect.CreateEffect(c)
+	e3:SetType(EFFECT_TYPE_FIELD)
+	e3:SetCode(EFFECT_CANNOT_ATTACK)
+	e3:SetRange(LOCATION_PZONE)
+	e3:SetTargetRange(LOCATION_MZONE,LOCATION_MZONE)
+	e3:SetTarget(c100217005.atktg)
+	c:RegisterEffect(e3)
+	--act limit
+	local e4=Effect.CreateEffect(c)
+	e4:SetType(EFFECT_TYPE_FIELD)
+	e4:SetCode(EFFECT_CANNOT_ACTIVATE)
+	e4:SetProperty(EFFECT_FLAG_PLAYER_TARGET)
+	e4:SetRange(LOCATION_PZONE)
+	e4:SetTargetRange(1,1)
+	e4:SetValue(c100217005.limval)
+	c:RegisterEffect(e4)
+	--double atk
+	local e5=Effect.CreateEffect(c)
+	e5:SetType(EFFECT_TYPE_FIELD)
+	e5:SetCode(EFFECT_SET_ATTACK_FINAL)
+	e5:SetRange(LOCATION_PZONE)
+	e5:SetTargetRange(LOCATION_MZONE,0)
+	e5:SetCondition(c100217005.atkcon0)
+	e5:SetValue(c100217005.atkval)
+	c:RegisterEffect(e5)
+	local e6=e5:Clone()
+	e6:SetTargetRange(0,LOCATION_MZONE)
+	e6:SetCondition(c100217005.atkcon1)
+	c:RegisterEffect(e6)
+end
+function c100217005.pencon(e,tp,eg,ep,ev,re,r,rp)
+	return eg:IsExists(Card.IsControler,1,nil,tp)
+end
+function c100217005.pentg(e,tp,eg,ep,ev,re,r,rp,chk)
+	if chk==0 then return Duel.CheckLocation(tp,LOCATION_SZONE,6) or Duel.CheckLocation(tp,LOCATION_SZONE,7) end
+end
+function c100217005.penop(e,tp,eg,ep,ev,re,r,rp)
+	if e:GetHandler():IsRelateToEffect(e)
+		and (Duel.CheckLocation(tp,LOCATION_SZONE,6) or Duel.CheckLocation(tp,LOCATION_SZONE,7)) then
+		Duel.MoveToField(e:GetHandler(),tp,tp,LOCATION_SZONE,POS_FACEUP,true)
+	end
+end
+function c100217005.splimit(e,c,tp,sumtp,sumpos)
+	return c:IsLocation(LOCATION_EXTRA) and bit.band(sumtp,SUMMON_TYPE_PENDULUM)==SUMMON_TYPE_PENDULUM
```

Comment:
```return not c:IsLocation(LOCATION_EXTRA) and bit.band(sumtp,SUMMON_TYPE_PENDULUM)==SUMMON_TYPE_PENDULUM```
because it should be except from the extra